### ISSUE TITLE
How to convert HTML style tags inside fpdf paragraphs?

### ISSUE THREAD

**Issue by me-suzy** [PyFPDF/fpdf2#498]
Hi!

I have several HTML tags inside the paragraphs I'm writing to the pdf file.
Those HTML tags contain some style I want to apply to the text I am writing. 
Let's consider this example:

My name is Emily. `<em>Hello world!<em>` My name is Jon.  (**in pdf file**)

Normally, on the HTML page this text is displayed like this (`<em>` tags transform the text into italic): 

My name is Emily.  _Hello world!_ My name is Jon. (**on HTML page**)

Is there any way to annotate the text, such that it will be displayed italic in the pdf, instead of <em></em>?

![image](https://user-images.githubusercontent.com/2770489/185735716-2331b35f-a0c0-4418-b370-d997c8e7b3c3.png)

So, basicaly, I need to change the second style contained in a HTML tag. For example:

`<p class="soft">I love design very much because it represent <em>the month december</em> that its winter time.</p>`

**must be visible in adobe as:**

<p class="soft">I love design very much because it represent <em>the month december</em> that its winter time.</p>

**Comment by Lucas-C** [comment]
Hi @me-suzy 
You have spotted an actual bug of `fpdf2`, thank you for reporting it.
I'm working on a fix

**Comment by Lucas-C** [comment]
@all-contributors please add @me-suzy for bug

**Comment by Lucas-C** [comment]
PR #499 fixed this

You can test it by installing the latest `fpdf2` version from git:

    pip install git+https://github.com/PyFPDF/fpdf2.git@master

**Comment by Lucas-C** [comment]
This has been broken since at least v2.2.0 of `fpdf2`:
https://github.com/PyFPDF/fpdf2/blob/2.2.0/test/html/test_html_features.pdf

**Comment by me-suzy** [comment]
Also, I have already installed GIT:

pip install github.py

But when I try to install your extension , and I got this error:

![image](https://user-images.githubusercontent.com/2770489/186088996-52648a83-c926-4696-9d18-0773cd0dd896.png)

**Comment by Lucas-C** [comment]
@me-suzy: no need to install `github.py`! This is a totally unrelated Pypi package: https://pypi.org/project/github.py/

You need to have the `git` command available, that is correct, but then just run this:

    pip install git+https://github.com/PyFPDF/fpdf2.git@master

This is syntax to install a Python package from a `git` repository.
Full documentation is available here: https://pip.pypa.io/en/stable/topics/vcs-support/

**Comment by me-suzy** [comment]
In the error it says that you have not installed git.
You need to install git.

[https://phoenixnap.com/kb/how-to-install-git-windows](https://phoenixnap.com/kb/how-to-install-git-windows)

You do these steps to Launch Git GUI (exclusive).

Open Git Bash.

Then go to the Configure GitHub Credentials step and run the 2 commands with your GitHub account data.

After doing that you run the command given by those guys.

And then you run the PDF converter.

**Comment by me-suzy** [comment]
Yes, I installed everything successfully, but it still looks the same.

This is the html paragraph with `<p class="text_obisnuit2"></p>` and inside this tag I have other 2 italic tags `<em></em>`

The basic tag doesn't see in PDF, very good. But the italic tag `<em></em>` is seen in PDF, and this is not good.

`<p class="text_obisnuit2"><em>Pentru a cunoa&#351;te realitatea un lider trebuie s&#259; de&#355;in&#259; &#351;i arta disimul&#259;rii &ndash; o arm&#259; de temut, dar eficient&#259; &icirc;n cele mai multe situa&#355;ii.</em></p>`

See the PDF after running Python code:

![image](https://user-images.githubusercontent.com/2770489/186769592-14567d7d-4678-4933-8dce-061d816998b1.png)

**Comment by Lucas-C** [comment]
The following code works perfectly fine for me with the latest version of `fpdf2`:
```python
from fpdf import fpdf, html

class PDF(fpdf.FPDF, html.HTMLMixin):
    pass

pdf = PDF()
pdf.add_page()
pdf.add_font("Kanit", fname="fonts/Kanit-Regular.ttf")
pdf.add_font("Kanit", style="I", fname="fonts/Kanit-Italic.ttf")
pdf.set_font("Kanit", size=24)
pdf.write_html('<p class="text_obisnuit2"><em>Pentru a cunoa&#351;te realitatea un lider trebuie s&#259; de&#355;in&#259; &#351;i arta disimul&#259;rii &ndash; o arm&#259; de temut, dar eficient&#259; &icirc;n cele mai multe situa&#355;ii.</em></p>')
pdf.output("issue_498.pdf")
```
The Kanit font is a Google font: https://fonts.google.com/specimen/Kanit

The resulting PDF contains text in italics: [issue_498.pdf](https://github.com/PyFPDF/fpdf2/files/9428595/issue_498.pdf)

**Comment by me-suzy** [comment]
I copy/paste my complete code on the link below. I also write the errors that I encountered

https://github.com/PyFPDF/fpdf2/issues/505#issuecomment-1228158345

**Comment by Lucas-C** [comment]
The fix from PR #499 has been released in [v2.5.7](https://github.com/PyFPDF/fpdf2/releases/tag/2.5.7)
